webui/react/src/pages/F_ExpList/glide-table/GlideTable.tsx

```diff
@@ -152,6 +152,10 @@ export const GlideTable: React.FC<GlideTableProps> = ({
     rows: CompactSelection.empty(),
   });
 
+  // Detect if user just click a row away from current selected group.
+  // If this stand alone select is set, use it as the base when doing multi select.
+  const [standAloneSelect, setStandAloneSelect] = React.useState<number>();
```

Comment:
prefer `singleSelection` since it matches `CompactSelection.fromSingleSelection`?